src/main/java/com/academy/edge/studentmanager/dtos/StudentResponseDTO.java

```diff
@@ -11,9 +11,12 @@
 public class StudentResponseDTO {
     private String id;
     private String name;
-    private String about;
-    private String photoUrl;
-    private String linkedIn;
+    private String birthdate;
     private Course course;
+    private String registration;
+    private String phone;
+    private String secondaryPhone;
     private String period;
+    private String entryPeriod;
+
```

Comment:
Adicionar dtype (descriminador de instrutor/student que está na class User), email e entryDate, apenas adicionar o modelMapper já faz o mapeamento.

```suggestion
    private String dtype;
    private String email;
    private String entryDate;
```